core/client/fs/src/main/java/alluxio/client/block/stream/LocalFilePacketReader.java

```diff
@@ -150,6 +154,10 @@ public void close() throws IOException {
       if (mClosed) {
         return;
       }
+      if (mReader != null) {
+        Preconditions.checkState(!mReader.isClosed() && mReader.getUsageCount() == 0);
```

Comment:
I think you can omit these checks since we already check the usage count in the reader's close method, and the `close` api is fine being called multiple times. Most close methods immediately return if it has already been closed instead of throwing an exception.